src/main/java/com/hyperwallet/clientsdk/model/HyperwalletPaperCheck.java

```diff
@@ -19,6 +19,8 @@
     private HyperwalletTransferMethod.Type type;
     private String token;
     private HyperwalletTransferMethod.Status status;
+    private HyperwalletTransferMethod.Status transition;
```

Comment:
The comment with respect to transition is generic. Pls fix in all the places.

Reply:
These were not refactored to keep it in accordance with V4 and the existing approach taken to refer to HyperwalletTransferMethod class for other fields like type and status. Changing the approach will call for changes in V4 and also other methods where HyperwalletPaperCheck is called.